### ISSUE TITLE
Bug: Wrong syntax highlighting for Terragrunt "inputs"

### ISSUE THREAD

**Issue by trallnag** [4ops/vscode-language-terraform#7]
Example shows it best (v0.2.1):

Here is a `terragrunt.hcl` file:

```hcl
remote_state {
  backend = "s3"

  config = {
    encrypt        = true
    bucket         = "tfstate-for-infrastructure-in-${local.environment_vars.account_id}"
    accesslogging_bucket_name = "tfstate-log-for-infrastructure-in-${local.environment_vars.account_id}"
    key            = "${path_relative_to_include()}/terraform.tfstate"
    region         = local.environment_vars.region
    dynamodb_table = "terraform-locks"
    s3_bucket_tags = local.global_vars.tags
    dynamodb_table_tags = local.global_vars.tags
  }

  generate = {
    path      = "backend.tf"
    if_exists = "overwrite_terragrunt"
  }
}

inputs = merge(
  local.account_vars.locals,
  local.region_vars.locals,
  local.environment_vars.locals,
)
```

Here is how this section looks in VS Code:

![image](https://user-images.githubusercontent.com/24834206/107966854-b49a4800-6fac-11eb-9efe-dfb8eb01ac34.png)

**Comment by trallnag** [comment]
Interestingly this is only highlighted incorrectly if a function is used like `merge`. If I replace it with a map it works fine:

```hcl
inputs = {
  account_id = local.environment_vars.account_id
  account_name = local.environment_vars.account_id
  region = local.environment_vars.account_id
}
```

![Uploading image.png…]()

**Comment by phantomkingx** [comment]
Not sure on complexity of fix but it would soothe my soul to see this fixed.

**Comment by bcbrockway** [comment]
We see a similar issue with different blocks:
```
include {
  path = find_in_parent_folders()
}

terraform {
  # source = "git::ssh://git@gitlab.com/mintel/satoshi/infrastructure/aws-infrastructure-modules//modules/services/eks-cluster/control-plane?ref=master"
  source = "../../../../../../../aws-infrastructure-modules//modules/services/eks-cluster/control-plane"
}

locals {
  account_vars     = read_terragrunt_config(find_in_parent_folders("account.hcl"))
  cluster_vars     = read_terragrunt_config(find_in_parent_folders("cluster.hcl"))
  common_vars      = read_terragrunt_config(find_in_parent_folders("common.hcl"))
  environment_vars = read_terragrunt_config(find_in_parent_folders("environment.hcl"))

  # Common Variables - These should stay the same for all clusters
  account_name          = local.account_vars.locals.account_name
  account_id            = local.common_vars.locals.account_ids[local.account_name]
  aws_auth_merger_image = local.common_vars.locals.aws_auth_merger_image
  cluster_type          = local.cluster_vars.locals.type
  environment_name      = local.environment_vars.locals.environment_name
  infra_tags            = local.common_vars.locals.tagsets.infra
}

dependencies {
  paths = ["../route53"]
}

dependency "vpc" {
  config_path = "../../../networking/vpc"
  mock_outputs = {
    vpc_id                 = "mock-vpc-id"
    vpc_cidr_block         = "1.2.3.4/20"
    private_app_subnet_ids = ["mock-subnet-id-priv-app"]
  }
  mock_outputs_allowed_terraform_commands = ["init", "validate"]
}

inputs = {

  #
  # Default Config: Same for all clusters (values pulled from elsewhere in this repo)
  #

  aws_account_name           = local.account_name
  aws_auth_merger_image      = local.aws_auth_merger_image
  custom_tags_eks_cluster    = local.infra_tags
  custom_tags_security_group = local.infra_tags
  eks_cluster_name           = local.environment_name
  eks_cluster_type           = local.cluster_type

  # We deploy EKS into the App VPC, inside the private app tier.
  vpc_id                       = dependency.vpc.outputs.vpc_id
  vpc_control_plane_subnet_ids = dependency.vpc.outputs.private_app_subnet_ids
  vpc_worker_subnet_ids        = dependency.vpc.outputs.private_app_subnet_ids

  #
  # Custom Config
  #
  kubernetes_version = "1.19"

  iam_role_to_rbac_group_mappings = {
    "arn:aws:iam::${local.account_id}:role/applied-innovation-ops" = ["applied-innovation-ops"]
    "arn:aws:iam::${local.account_id}:role/cpg-ops"                = ["cpg-ops"]
    "arn:aws:iam::${local.account_id}:role/dev-tools-ops"          = ["dev-tools-ops"]
    "arn:aws:iam::${local.account_id}:role/portal-ops"             = ["portal-ops"]
  }

  num_nat_gateways = 1

  # Use the latest patch release that matches the minor k8s release version you're using (check
  # https://github.com/kubernetes/autoscaler/releases)
  cluster_autoscaler_version = "v1.19.1"
  
  enable_idp = false

}
```

looks like:

![image](https://user-images.githubusercontent.com/6728173/117947443-1b0ed800-b308-11eb-9cc1-5f8c653bd34d.png)

**Comment by stephenteodori** [comment]
Also seeing error highlighting for `terraform_version_constraint` and `terragrunt_version_constraint`.

```
terraform_version_constraint = "0.14.11"
terragrunt_version_constraint = ">= 0.29.0"
```
![image](https://user-images.githubusercontent.com/1486375/126203428-d7d7e6b0-f516-4d2e-a3d1-93cc8206d990.png)

**Comment by 4ops** [comment]
Fixed in [v0.2.5](https://github.com/4ops/vscode-language-terraform/releases/tag/v0.2.5)
